clusterloader2/pkg/flags/flags.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flags
+
+import (
+	"flag"
+	"os"
+
+	"github.com/spf13/pflag"
+)
+
+func init() {
+	pflag.CommandLine = pflag.NewFlagSet(os.Args[0], pflag.ContinueOnError)
+	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
+}
+
+var flags []flagEnv
+
+// StringVar creates string flag with given parameters.
+func StringVar(s *string, flagName, defaultValue, description string) {
+	pflag.StringVar(s, flagName, defaultValue, description)
+}
+
+// IntVar creates int flag with given parameters.
+func IntVar(i *int, flagName string, defaultValue int, description string) {
+	pflag.IntVar(i, flagName, defaultValue, description)
+}
+
+// StringEnvVar creates string flag with given parameters.
+// If flag is not provided, it will try to get env variable.
+func StringEnvVar(s *string, flagName, envVariable, defaultValue, description string) {
+	*s = defaultValue
+	stringFlag := &stringFlagEnv{
+		v:           s,
+		envVariable: envVariable,
```

Comment:
Why do we need to pass envVariable to the bottom?
I would say it's cleaner to say:

*s = ParseStringEnv(envVariable, defaultValue)
stringFlag := &stringFlagEnv{v: s}
pflag.Var(stringFlag, flagName, description)